### PR TITLE
Deployment Process Documentation

### LINKED ISSUE #2: Compatibility with PDS4 version 1.13.0.0
Hi there,

I've just taken PLAID for a spin at it seems to be a really nifty tool. However, I am wondering whether there are plans to update PLAID to support v1.13 of the information model? If not, could you provide a rough estimate of the work required to complete such work?

All the best,
Ariel Ladegaard

### PR DESCRIPTION
**Pull request by voxparcxls**
### Added:
* deployment-docs/DEV-DEPLOY.md
* deployment-docs/PROD-DEPLOY.md

### Changes:
Set hostname variable to `HTTP_HOST` to update account verification and password reset e-mail messages.
* php/resend_verification_email.php
* php/interact_db.php

## ♻️ Related Issues

https://github.com/NASA-PDS/PLAID/issues/16 
https://github.com/NASA-PDS/PLAID/issues/26

### CODE REVIEW

**Comment by c-suh** [comment]
Did review/run-through with Josh via call. He will make updates and mention me in a comment when done and this is ready to review again.

**Comment by voxparcxls** [comment]
@jordanpadams @c-suh  

The document is updated based on the review and recommended changes. 

Files changed:
- deployment-docs/DEV-DEPLOY.md (updated since review)
- docker-compose.dev.yaml (added docker image name and container's DB mounting)

File: deployment-docs/DEV-DEPLOY.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+## Docker Dev Deployment
+
+### _** Prerequisites before proceeding with deployment instructions_
+
+1. Required: Access to JPL MIPL Account and `sudo` user privileges for the virtual machine: _`miplvm-docker1`_
```
Comment (jordanpadams) [inline deployment-docs/DEV-DEPLOY.md:5]: @voxparcxls I will merge this for now, but we can talk about this next week. This is not possible for the users of this application. There has to be another way we can build and deploy the docker image on my laptop.

**Review by jordanpadams — APPROVED** [review]

**Comment by jordanpadams** [comment]
@voxparcxls can you please resolve the conflicts on this repo? Looks like you may have been using an old branch for updating these files.

the easiest may be to create a new branch from main and just add the 2 new markdown files.

**Comment by jordanpadams** [comment]
closed for #29

**Pull request closed without merging**